frappe/core/doctype/prepared_report/prepared_report.json

```diff
@@ -42,16 +42,10 @@
    "fieldname": "column_break_4",
    "fieldtype": "Column Break"
   },
-  {
-   "fieldname": "report_start_time",
```

Comment:
This was quite misleading then :pensive: wait time isn't accounted for here. Anyway probably doesn't matter much.